log-patch.diff

```diff
@@ -0,0 +1,129 @@
+diff --git a/config.h b/config.h
```

Comment:
![NO!!!](https://media.giphy.com/media/12XMGIWtrHBl5e/giphy.gif)

Reply:
Forgot to delete that.